client/src/main/java/com/cyberark/conjur/sdk/ApiClient.java

```diff
@@ -1302,7 +1304,9 @@ public AccessToken getNewAccessToken() {
             ApiResponse<String> response = execute(c, new TypeToken<String>(){}.getType());
             accessToken = response.getData();
         } catch (ApiException e) {
-            System.out.println("Failed to automatically update AccessToken");
+            LOGGER.severe("Status code: " + e.getCode());
+            LOGGER.severe("Reason: " + e.getResponseBody());
+            LOGGER.severe(e.getMessage());
```

Comment:
why are we not logging actual stack trace?

Reply:
Logging the stack trace seemed to be like a bonus. 
These informations are the minimal information required to troubleshoot.
But definitely agree that nothing better than logging the stackTrace.